pkg/env/reprocessor.go

```diff
@@ -0,0 +1,6 @@
+package env
+
+var (
+	// ReprocessorSemaphoreLimit is the maximum number of reprocessed images.
+	ReprocessorSemaphoreLimit = RegisterIntegerSetting("ROX_REPROCESSOR_SEMAPHORE_LIMIT", 10)
```

Comment:
Hmm, I wonder if we should be more "aggressive" and bump it to the same amount as https://github.com/stackrox/stackrox/blob/master/pkg/env/max_parallel_scans.go; but this probably does warrant looking at the scale tests more to see the effect of it, so feel free to ignore.

Reply:
Let's increase it gradually. I also think that we can likely go to something like 30 images based on the memory profile of our big cloud tenants (barely any reprocessor spike currently).